### ISSUE TITLE
Build instructions?

### ISSUE THREAD

**Issue by ryanfantus** [pylgrym/basin#1]
Hi, I'd like to try this game, do you have build instructions? I'm using linux. Thanks!

**Comment by pylgrym** [comment]
Yep, I'll put them in the readme.md
(the reason they are 'missing' are because they are in video#2).

I'll send a second message once I've fixed it, I'm at work.
Cheers, Jakob.

**Comment by jgaardsted** [comment]
> Hi, I'd like to try this game, do you have build instructions? I'm using linux. Thanks!

Hi again. I just looked at it, the 'instructions' are pretty simple:
It is simply a Visual Studio Windows C++ project,
so it should build just by opening the .csproj project file in Visual Studio, build and run it directly.
I'll try to build it later tonight, to see if it still runs.
I think the year was VS2015 or VS17, when I first wrote it.
Cheers, Jakob.
